src/app/organizations/collections/collections.component.html

```diff
@@ -64,6 +64,20 @@ <h4>{{ collection.value.displayName }}</h4>
                   <span *ngIf="collection.value.toolsLength !== 1; else oneTool"> Tools</span>
                   <ng-template #oneTool> Tool</ng-template>
                 </span>
+                <span class="bubble" data-cy="collections-services-count-bubble" *ngIf="collection.value.servicesLength > 0">
+                  {{ collection.value.servicesLength }}
+                  <span *ngIf="collection.value.servicesLength !== 1; else oneService"> Services</span>
+                  <ng-template #oneService> Service</ng-template>
+                </span>
```

Comment:
Please don't change, but someday, we might consider implementing a `numbered` pipe, that takes a number and some arguments: arg1 is the string to be numbered, and optional arg2 is the plural form of arg1 (default value is arg1 + "s").   Then, we can replace some of the above code with something like:
```
{{ collection.value.servicesLength | numbered:"Service" }}
```